ci/test/cargo-test/mzcompose.py

```diff
@@ -89,6 +90,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             "cargo",
             "llvm-cov",
             "nextest",
+            "--all-features",
```

Comment:
@def- Updated here and below, lmk if this is not right.

Reply:
No complaints from my side